docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,397 @@
+
+### 8. Start using Ibexa DXP
+
+You can now start using Ibexa DXP and implement your own website on the platform.
+
+You can edit the configuration and code in the DDEV project directory.
+You can use commands listed in the documentation by prefixing them with `ddev exec` or by opening a terminal inside the container using `ddev ssh`.
+For example, if a guideline invites you to run `php bin/console cache:clear`, you can do it in the DDEV container in one of the following ways:
+
+- run `ddev php bin/console cache:clear`
+- enter `ddev ssh` and run `php bin/console cache:clear` after the new prompt
+
+## Other options for configuration
+
+DDEV offers several ways to achieve a same thing, offering different levels of flexibility or adaptability to your development environment.
+
+!!! tip
+
+    Learn more about the [DDEV commands](https://ddev.readthedocs.io/en/latest/users/usage/commands/):
+    
+    - by running [`ddev list`](https://ddev.readthedocs.io/en/latest/users/usage/commands/#list) to list them all,
+    - by running [`ddev help <command>`](https://ddev.readthedocs.io/en/latest/users/usage/commands/#help) to get usage details about a given command.
+    
+    Learn more about DDEV configuration at [`ddev config` command documentation](https://ddev.readthedocs.io/en/latest/users/usage/commands/#config) and [advanced configuration files documentation](https://ddev.readthedocs.io/en/latest/users/configuration/config/).
+
+
+### Using `auth.json`
+
+An `auth.json` file can be used for one project, or globally for all projects, with the [DDEV `homeaddition` feature](https://ddev.readthedocs.io/en/latest/users/extend/in-container-configuration/).
+
+For example, you can copy an `auth.json` file to a DDEV project:
+`cp <path-to-an>/auth.json .ddev/homeadditions/.composer`
+
+Alternatively, the Composer global `auth.json` can be the DDEV global `auth.json` with the help of symbolic link:
+`mkdir -p ~/.ddev/homeadditions/.composer && ln -s ~/.composer/auth.json ~/.ddev/homeadditions/.composer/auth.json`
+
+If the DDEV project has already been started, you need to run `ddev restart`.
+
+To use an `auth.json` file replaces the step [4. Composer authentication](#4-composer-authentication).
+
+### Using Dotenv
+
+Instead of using environment variables inside the container, a [`.env.local`](https://symfony.com/doc/5.4/configuration.html#overriding-environment-values-via-env-local) file can be added to the project.
+
+The following shows `.env.local` configuration on the example of the database:
+
+- Skip step [2. Configure DDEV / Configure database connection](#configure-database-connection).
+- Modify step [5. Create Ibexa DXP project](#5-create-ibexa-dxp-project) to insert the database setting (Commerce and MariaDB are used in this example):
+  ```bash
+  ddev composer create ibexa/commerce-skeleton --no-install;
+  echo "DATABASE_URL=mysql://db:db@db:3306/db" >> .env.local;
+  ddev composer install;
+  ```
```

Comment:
@adriendupuis  I think it's worth mentioning about this earlier, around this 2nd step which has to be skipped in the beginning of the procedure.

Reply:
Good remark. Mentioned in "Configure database connection" (addition in 238969ccb6ea437e9391493ff049c93aaf2ee665)